### DISCUSSION TITLE
Post process problem in example '4_restart'

### DISCUSSION THREAD

**Discussion by kwche** [neperfepx/FEPX#31] [Q&A] [answered]
After running **post_process.sh**,  output images can not be produced. It seems that the command **neper -S cycle1.sim,cycle2.sim -o cycle1-2.sim** didn't work, because in directory **cycle1-2.sim** there are no **slip** results. Strangely **coo**  appears alone in every result directory, as shown below. Could you please tell me how to solve this problem?

kawai@kawai-virtual-machine:~/examples/4_restart/cycle1-2.sim/results/elements$ ls
coo

kawai@kawai-virtual-machine:~/examples/4_restart/cycle1-2.sim/results/elsets$ ls
coo

kawai@kawai-virtual-machine:~/examples/4_restart/cycle1-2.sim/results/nodes$ ls
coo

**Comment by rquey** [comment, marked as answer]
Please update your Neper to the latest version (https://github.com/rquey/neper, version `4.2.1-3`) and try again.

**Reply by kwche** [reply]
It works! No problem in new version. Thank you!
